code/modules/projectiles/projectile/explosive.dm

```diff
@@ -9,22 +9,30 @@
 	does_spin = 0
 
 /obj/item/projectile/bullet/srmrocket/on_hit(atom/target, blocked=0)
-	..()
```

Comment:
Removing the parent call seems a bit odd. The super call is where various affects are applied and it shouldn't impact the following code.

Reply:
As far as I know the only effect this projectile needs to apply is the explosion.